stable/syncthing/SCALE/questions.yaml

```diff
@@ -141,17 +141,120 @@ questions:
                       schema:
                         type: int
                         default: 8384
-                        editable: false
+                        editable: true
+                        hidden: false
```

Comment:
This is not needed at all. You're not going to connect other services to this and even if you do, it doesn't have to be any other port anyway.

This might become relevant once we start supporting servicetype: loadbalancer, but thats not the case today.

Reply:
As in, the entire internal service port section in unnecessary?

Reply:
No, you made it editable and unhidden, which you shouldn't.

Reply:
Ah right I misunderstood. I made it editable because technically it also is editable in the web-ui settings, but there really is no reason for wanting to change this afaik.

Reply:
Thats not true, syncthing has no effect on the "targetPort" setting, this is the internal port that's used by the k8s service. That has nothing to do with syncthing.